src/player.cpp

```diff
@@ -5661,16 +5661,18 @@ void player::try_to_sleep( const time_duration &dur )
     bool watersleep = false;
     if( has_trait( trait_CHLOROMORPH ) ) {
         plantsleep = true;
-        if( ( ter_at_pos == t_dirt || ter_at_pos == t_pit ||
-              ter_at_pos == t_dirtmound || ter_at_pos == t_pit_shallow ||
-              ter_at_pos == t_grass ) && !vp &&
-            furn_at_pos == f_null ) {
-            add_msg_if_player( m_good, _( "You relax as your roots embrace the soil." ) );
-        } else if( vp ) {
+        // Down to bottom: if in vehicle - can't sleep
+        if( vp ) {
```

Comment:
This has the opposite effect of the comment.
```suggestion
        if( !vp ) {
```